app/controllers/vendors_controller.rb

```diff
@@ -0,0 +1,30 @@
+class VendorsController < ApplicationController
```

Comment:
I think codacy is complaining because there's no authorization check on this controller - see items controller for example, it will have these lines:

```
  before_filter :authenticate_user!
  load_and_authorize_resource
```

need to update ability.rb to give existing roles the ability to edit vendors.

ref: https://github.com/CanCanCommunity/cancancan/wiki/authorizing-controller-actions

Reply:
done!